README.md

```diff
@@ -12,6 +12,11 @@ _1 Chai = 1 Dai * Pot.chi_
 
 You can interact with the Chai contract at [chai.money](https://chai.money). The source for the ui is hosted [here](https://github.com/lucasvo/chui).
 
+## audit
```

Comment:
```suggestion
## Audit
```

Reply:
lower case is in line with other headers